### ISSUE TITLE
Questions about the training data

### ISSUE THREAD

**Issue by XiaoqiangZhou** [NVlabs/imaginaire#19]
Thanks for your such a great series of works on image and video synthesis.

I'm very interested in the work "World-Consistent Video-to-Video Synthesis", which solves the long-term visual consistency in video-to-video synthesis efficiently. 

I hope to re-training models of the work and be more familiar with this area. However, I don't know how to deal with the data preparation, which consists of many steps, like edge map, depth map, segmentation label, and they are related to many tools/SOTA methods. Could you please give more details of these data preparation steps? Personally, I think it will help people enter this task and contribtue the growth of this community.

Thanks.

**Comment by arunmallya** [comment]
Hi, I will try to release the toolkit for data preparation as soon as possible. Am a bit busy due to upcoming deadlines.
Here are some resources used, in case you want to get started:
Semantic segmentation - https://github.com/NVIDIA/semantic-segmentation
MegaDepth depth maps - https://github.com/zhengqili/MegaDepth
HED edge maps - https://github.com/sniklaus/pytorch-hed
DensePose - https://github.com/facebookresearch/detectron2
OpenPose - https://github.com/CMU-Perceptual-Computing-Lab/openpose

As for unprojections, will update here: https://github.com/NVlabs/imaginaire/issues/5

**Comment by XiaoqiangZhou** [comment]
Thanks for your help! I will try to start with these resources. Also, I'm looking forward to your toolkit release.

**Comment by razimgit** [comment]
@arunmallya  Any updates on the toolkit?

I wasn't successful with #5 generating unprojections. Ideally, I'd want to use a depth map that I already have from a simulator, rather than generating points with OpenSfM. Could that be possible?

**Comment by siddharthbhushan** [comment]
> 

Hi @razimgit,

I am trying to do something similar. Did you have any success using your own depth map?

**Comment by razimgit** [comment]
@siddharthbhushan unfortunately, no.
